Ginger/GingerCoreNET/Logger/WebReportGenerator.cs

```diff
@@ -215,12 +215,14 @@ private void PopulateMissingFields(LiteDbRunSet liteDbRunSet, string clientAppPa
                         }
                     }
                 }
+                liteDbRunner.RunStatus = liteDbRunner.SetStatus(liteDbRunner.BusinessFlowsColl);
```

Comment:
Need to check why the status is not updated during regular execution. We should not update it here explicitly